docs/tasks/administer-cluster/coredns.md

```diff
@@ -1,11 +1,11 @@
 ---
-approvers:
+reviewers:
```

Comment:
Oops, I didn't mean to make this change. What should this be? I am not in the K8s org, so I can't actually approve it.

Reply:
@johnbelamaric can you just change this value back using a second commit?

Reply:
will do